lib/ecto_shorts/actions.ex

```diff
@@ -827,31 +827,7 @@ defmodule EctoShorts.Actions do
     {status, Enum.reverse(res)}
   end
 
-  defp repo!(opts) do
-    with nil <- repo(opts) do
-      raise ArgumentError, message: "ecto shorts must be configured with a repo. For further guidence consult the docs. https://hexdocs.pm/ecto_shorts/EctoShorts.html#module-config"
-    end
+  defp default_opts do
+    [repo: Config.repo(), replica: Config.replica()]
```

Comment:
I think we're missing the error functionality now, we should check the values to ensure they're not nil